content/en/servers/blocks/put/index.md

```diff
@@ -0,0 +1,25 @@
++++
+title = '📨 PUT'
+headless = true
+time = 20
+facilitation = false
+emoji= '🧩'
+[objectives]
+    1='Process a PUT request using Express and Node and update an array held in memory'
++++
+
+> `PUT /movies/:movieId should update a movie (that matches the passed movieId)`
+
+This means that `PUT /movies/2` should update an movie with the id `2` and return `200` with `JSON { success: true }` to the user.
+
+The code should look something like this:
```

Comment:
I think we should either say this is starter code, or fill out the example below to demonstrate the actual handling of updating a movie in the snippet.

I think I'd lean towards the former:

```suggestion
To get you started, the code for handling a `PUT` request is here:
```